src/pybind/mgr/dashboard/frontend/src/app/ceph/shared/device-list/device-list.component.html

```diff
@@ -13,6 +13,7 @@
 
 <ng-template #lifeExpectancy
              let-value="value">
+  <span *ngIf="!value.life_expectancy_enabled" i18n>{{ "" | notAvailable }}</span>
```

Comment:
lint will complain about this one!

```suggestion
  <span *ngIf="!value.life_expectancy_enabled"
         i18n>{{ "" | notAvailable }}</span>
```